src/main/resources/assets/schemas/Dac.yaml

```
@@ -12,6 +12,9 @@ properties:
   email:
     type: string
     description: Optional DAC Email address
+  associatedDaa:
+    type: string
```

I think this should be:
```suggestion
    $ref: './DataAccessAgreement.yaml'
```
See `LibraryCard.yaml` for a working example - we have some other examples that are unfortunately incorrect